README.md

```diff
@@ -73,3 +73,20 @@ We are doing this to be able to troubleshoot the viz while the model is still ru
     ```
 
 You should now have a set of subset NetCDF files named with the appropriate date in your environment.
+
+
+## Updating the Docker images
+
+There are two docker images used in the processing steps --- one for R, and one for tippecanoe.  The built images are stored in the Docker registry at https://code.chs.usgs.gov/wma/iidd/wbeep-data-processing.  If you need to update the images, follow these steps:
+
+1. Make the appropriate changes to the Dockerfile, and make sure it builds locally by running `docker-compose build` in the terminal from the directory containing both the Dockerfile and docker-compose.yml.  
+
+2. Make a pull request and have it reviewed.  
+
+3. When the pull request is merged, it should trigger a Jenkins job (watching only the Dockerfile in the repo) that will build the image and push it to the docker registry tagged as `R-latest` or `tippecanoe-latest`.
```

Comment:
doesn't seem like this job is triggered based on merges into the wbeep-processing repo right now, but I could change that if we want that to be monitoring the repo?

Reply:
actually neither, i'll set them up to poll

Reply:
Cool.  We'll probably just want to revisit this if we decide to start uniquely tagging images